nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -212,19 +201,17 @@ void InflationLayer::updateCosts(
   unsigned char * master_array = master_grid.getCharMap();
   unsigned int size_x = master_grid.getSizeInCellsX(), size_y = master_grid.getSizeInCellsY();
 
-  if (seen_ == NULL) {
+  if (seen_.empty()) {
```

Comment:
I think you can get rid of this check now. If `seen_.empty()` then also `seen_.size() != size_x*size_y` unless one of the sizes is `0`. I expect somewhere else in the code should be the check to ensure the sizes aren't 0, so I think we can leave that out here.

Reply:
Ok, pushed a change.